nerfstudio/models/base_surface_model.py

```diff
@@ -292,7 +292,7 @@ def get_loss_dict(self, outputs, batch, metrics_dict=None) -> Dict[str, torch.Te
             metrics_dict: dictionary of metrics, some of which we can use for loss
         """
         loss_dict = {}
-        image = batch["image"].to(self.device)
+        image = batch["image"][..., :3].to(self.device)
```

Comment:
Shouldn't this also call blend_background?

Reply:
This would just add an additional function call to every model that does not use alpha transparent training. Is there a reason why you would prefer using `blend_background` here? I think it would only be more confusing if we call `blend_background(rgb=batch["image"][..., :3].to(self.device), opacity=None, background_color=None)` instead of `batch["image"][..., :3].to(self.device)`. I passed the optional arguments for clarity.

Reply:
What I meant was: shouldn't this be implemented for all methods - not just nerfacto?

Reply:
That would be an option. I didn't do it based on the suggestion [here](https://github.com/nerfstudio-project/nerfstudio/pull/2025#issuecomment-1601654847). But it wouldn't take much time to add it to the other currently existing models once we agree on the `nerfacto` implementation :)

Reply:
I think it would be really cool to have this implemented for all methods.